hw/ip_templates/ac_range_check/rtl/ac_range_check.sv.tpl

```diff
@@ -101,14 +113,21 @@ module ${module_instance_name}
     assign addr_hit[i] = prim_mubi_pkg::mubi4_test_true_loose(reg2hw.range_perm[i].enable.q) &
                          tor_hit;
 
+    // Perform RACL checks - check if the incoming role matches with the configured policy
+    assign racl_read_hit [i] =  |(racl_role_vec & reg2hw.range_racl_policy_shadowed[i].read_perm.q)
```

Comment:
Nit: There's an extra space here! Probably drop one before the `|` and keep the one before the `[`?